src/doc/rustdoc/src/lints.md

```diff
@@ -285,3 +285,47 @@ warning: unclosed HTML tag `h1`
 
 warning: 2 warnings emitted
 ```
+
+## automatic_links
+
+This link is **allowed by default** and is **nightly-only**. It detects links
```

Comment:
Any reason not to make this warn by default?

Reply:
Just the habit. :)